### ISSUE TITLE
Invalid bower fields.

### ISSUE THREAD

**Issue by CupOfTea696** [zeusdeux/isInViewport#26]
Got some warnings when installing via bower:

```
bower invalid-meta  The "name" is recommended to be lowercase, can contain digits, dots, dashes
bower invalid-meta  The "main" field cannot contain minified files
```

**Comment by zeusdeux** [comment]
Since I haven't had a lot of people use bower, I haven't kept up with newer version. A PR for this would be greatly appreciated though :)

**Comment by zeusdeux** [comment]
Fixed by #27. Thanks :)
